api/src/main/java/org/pmiops/workbench/db/model/DbWgsExtractCromwellSubmission.java

```diff
@@ -112,6 +116,33 @@ public void setCompletionTime(Timestamp completionTime) {
     this.completionTime = completionTime;
   }
 
+  @Column(name = "terra_status")
+  public Short getTerraStatus() {
```

Comment:
nit: I like to make this and the function below as private to discourage use of the Short version of the enum.

Reply:
Ooh good idea